src/main.ts

```diff
@@ -92,7 +92,7 @@ function isLabeledStale(issue: Issue, label: string): boolean {
 }
 
 function wasLastUpdatedBefore(issue: Issue, num_days: number): boolean {
-  const daysInMillis = 1000 * 60 * 60 * num_days;
+  const daysInMillis = 1000 * 60 * 60 * 24 * num_days;
```

Comment:
Perhaps it makes sense to adjust to

```suggestion
  const daysInMillis = num_days * 24 * 60 * 60 * 1000;
```

?